ui/pages/settings/networks-tab/networks-list-item/networks-list-item.js

```diff
@@ -68,7 +66,7 @@ const NetworksListItem = ({
         setSearchedNetworks([]);
         dispatch(setSelectedSettingsRpcUrl(rpcUrl));
         if (!isFullScreen) {
-          history.push(NETWORKS_FORM_ROUTE);
+          global.platform.openExtensionInBrowser(NETWORKS_ROUTE);
```

Comment:
Why `NETWORKS_ROUTE` instead of `NETWORKS_FORM_ROUTE`?

Reply:
```
const NETWORKS_ROUTE = '/settings/networks';
const NETWORKS_FORM_ROUTE = '/settings/networks/form';
```

Went down a bit of a rabbit hole here. As the [issue](https://github.com/MetaMask/metamask-extension/issues/15405) shows, the route, `settings/networks/form`,  resolves to the generic `settings` page. Why? It appears the path is not resolving to the networks tab properly and from there, the `shouldRenderNetworkForm` value when the route is `NETWORKS_FORM_ROUTE` is not being set. 

I think that applying the current changes now would make `NETWORKS_FORM_ROUTE` obsolete which might now be what we want.

Reply:
```
const NETWORKS_ROUTE = '/settings/networks';
const NETWORKS_FORM_ROUTE = '/settings/networks/form';
```

Went down a bit of a rabbit hole here. As the [issue](https://github.com/MetaMask/metamask-extension/issues/15405) shows, the route, `settings/networks/form`,  resolves to the generic `settings` page. From my investigation, it appears the path is not resolving to the networks tab properly and from there, the `shouldRenderNetworkForm` value when the route is `NETWORKS_FORM_ROUTE` is not being set. 

hardcoding the route to resolve to the networks tab and setting `shouldRenderNetworkForm` to true shows this popup page:  
<img width="396" alt="Screenshot 2022-08-03 at 8 53 17 PM" src="https://user-images.githubusercontent.com/20778143/182615194-53758684-4c0b-4a5d-b32d-67f85d298760.png">

I think that applying the current changes now would make `NETWORKS_FORM_ROUTE` obsolete which might now be what we want.

Reply:
```
const NETWORKS_ROUTE = '/settings/networks';
const NETWORKS_FORM_ROUTE = '/settings/networks/form';
```

Went down a bit of a rabbit hole here. As the [issue](https://github.com/MetaMask/metamask-extension/issues/15399) shows, the route, `settings/networks/form`,  resolves to the generic `settings` page. From my investigation, it appears the path is not resolving to the networks tab properly and from there, the `shouldRenderNetworkForm` value when the route is `NETWORKS_FORM_ROUTE` is not being set. 

hardcoding the route to resolve to the networks tab and setting `shouldRenderNetworkForm` to true shows this popup page:  
<img width="396" alt="Screenshot 2022-08-03 at 8 53 17 PM" src="https://user-images.githubusercontent.com/20778143/182615194-53758684-4c0b-4a5d-b32d-67f85d298760.png">

I think that applying the current changes now would make `NETWORKS_FORM_ROUTE` obsolete which might not be what we want.  

p.s. i deleted the last copy of this comment and reposted this comment with the proper reference to see if I can remove the reference to the incorrect issue ticket

Reply:
p.s.p.s. the reference to this PR from the unrelated issue was not removed when I deleted the comment I mentioned above